arbitrum-docs/for-devs/concepts/token-bridge/token-bridge-ether.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: 'Token bridge: Bridging ether'
+sidebar_label: Bridging ether
+description: Describes how bridging ether works on Arbitrum
+reader-audience: developers who want to build on Ethereum/Arbitrum
+content-type: concept
+sidebar_position: 2
+---
+
+import DepositEthDiagram from '@site/../arbitrum-docs/diagrams/_deposit-eth.mdx';
+import WithdrawEthDiagram from '@site/../arbitrum-docs/diagrams/_withdraw-eth.mdx';
+
+Ether (ETH) is the native currency of Ethereum, Arbitrum One and Arbitrum Nova. It is used to pay the necessary fees to execute a transaction in those chains. Bridging ETH from Ethereum (Layer 1, or L1) to an Arbitrum chain (Layer 2, or L2) follows, thus, a different process than the one followed when bridging other types of asset.
```

Comment:
Should we replace this w the following, since we are missing Arb Goerli?
Ether (ETH) is the native currency of Ethereum and all Arbitrum chains.